bare_header/sifive_hca_0_5_x.h

```diff
@@ -16,9 +16,6 @@ class sifive_hca_0_5_x : public Device {
       : Device(os, dtb, "sifive,hca-0.5.(\\d+)") {}
 
   void emit_defines() override {
-    emit_compat("sifive,hca");
```

Comment:
Actually, shouldn't you have this in the match regex below. so that it get emitted if there is a HCA.

Reply:
you ar right, but i did not use this define in any case so I have just remove it.